groups/instance/instance.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_instance" "artifactory" {
+  ami           = var.ami
```

Comment:
You're specifying a defaulted variable value for the AMI. This should be using the value from the `artifactory_ami` `aws_ami` data lookup

Reply:
Requires https://github.com/companieshouse/artifactory-ami/pull/1

Reply:
Does this mean you haven't created the AMI yet?